AstSemantics.md

```diff
@@ -534,6 +534,13 @@ Truncation from floating point to integer where IEEE 754-2008 would specify an
 invalid operation exception (e.g. when the floating point value is NaN or
 outside the range which rounds to an integer in range) traps.
 
+## Type-parameterized operators.
+
+  * `select`: a ternary operator with a boolean (i32) condition and two
+    additional operands with the parameterized type. `select` returns the
+    the first type-parameterized operand if the condition operand is non-zero,
+    or the second type-parameterized operand otherwise.
```

Comment:
I think it'd be simpler to not say "parameterized" at all and just have a sentence at the end saying the parameters must have the same value type.